packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Question looking at the snapshots - are we (or are we planning to) update the tooltips to say why certain controls are disabled?
![image](https://user-images.githubusercontent.com/2684369/184920065-5aaa30ee-baaf-4f97-999c-73ae7963ca71.png)

e.g. say something like `You do not have permission to start your camera in this call`

Reply:
If we do, we should add snapshot tests that capture the tooltip

Reply:
We had the discussion around that and we are not going to have specific tooltips for rooms as of now. If we add the tooltips in future, will add the tests for same :)